src/app/saga/manuscript.saga.ts

```diff
@@ -21,7 +25,9 @@ export function* loadManuscriptSaga(action: Action<string>) {
     try {
       const changes = yield call(getManuscriptChanges, id);
       manuscript = applyChangesFromServer(manuscript, changes);
-    } catch (e) {}
+    } catch (e) {
+      console.error('Loading changes failed', e);
```

Comment:
Probably do need to think about what we do when this happens in the real world. We'll need to inform the user somehow about the error.